docs2/site/docs/migrations/migration5.md

```diff
@@ -399,3 +399,10 @@ string s = writer.ToString();
 `SDLPrinter` is a highly optimized visitor for asynchronous non-blocking SDL output
 into provided `TextWriter`. In the majority of cases it does not allocate memory in
 the managed heap at all.
+
+### 21. More strict `InputObjectGraphType<TSourceType>.IsValidDefault`
+
+This method began to perform more checks of the specified value. Current implementation
+can cause problems in some situations. Changes are made for earlier error detection - at
+the schema initialization stage. If you encountered that your code stopped working,
+then feel free to post an issue.
```

Comment:
I reviewed everything again and the code is okay.  But the migration documentation needs to be rewritten.  This seems rather vague and does not indicate the most important point of the new code - ToAST is supported and enables printing a code-first schema that uses `InputObjectGraphType`.  We need not instruct the user to post an issue if they have a problem; let's just tell them what's changed and what they need to change in their code.

I would suggest that in the 'new features' section, you discuss support of `ToAST` and `IsValidDefault` and the fact that schemas with default input objects can be printed, while in the breaking changes you say that if you override `ParseDictionary`, they may now be required to also override `ToAST`.

Reply:
Ideally I would suggest a small snippet of the prior one-line `IsValidDefault` method to revert behavior if necessary, so they can cut and paste it if there's an issue in their situation.

Reply:
OK. I'll add notes tomorrow, 3 AM here.

Reply:
Done. Not sure about wordinds.

Reply:
Good enough.  Probably we will review entire migration document before releasing anyway. :+1: